web/src/main/ui/app/components/flows-new/edit-flow/ui/new-step-dialog-ui.component.ts

```diff
@@ -49,7 +49,7 @@ export class NewStepDialogUiComponent implements OnInit {
       description: [this.step ? this.step.description : ''],
       sourceQuery: [this.step ? this.step.options.sourceQuery : ''],
       sourceCollection: [this.step ? this.step.options.sourceCollection : ''],
-      targetEntity: [this.step ? this.step.options.targetEntity : ''],
+      targetEntity: [this.step ? this.step.options.targetEntity : '', Validators.required],
```

Comment:
This field is not shown on the Ingestion step. You need to add required Validator only if it is not ingestion...

Reply:
Thanks Sasha.  Caught the problem.

Reply:
This case is already handled by NewStepDialogValidator

`
  if (group.value.stepDefinitionType === this.stepType.MAPPING ||
      group.value.stepDefinitionType === this.stepType.MASTERING) {
    if (!group.value.targetEntity) {
      errors['noTargetEntity'] = true;
    }
`

Reply:
So I'm removing this standard validator